lsp-ui-doc.el

```diff
@@ -86,6 +86,16 @@ Child frames requires GNU/Emacs version >= 26 and graphical frames."
   :type 'boolean
   :group 'lsp-ui-doc)
 
+(defcustom lsp-ui-doc-use-webkit nil
+  "Whether to display documentation in a WebKit widget in a child-frame.
+This requires GNU/Emacs version >= 26 and built with the `--with-xwidgets`
+option."
+  :type 'boolean
+  :group 'lsp-ui-doc)
+
+(when lsp-ui-doc-use-webkit
```

Comment:
This will work only if you set `lsp-ui-doc-use-webkit` upfront, but you could change this value afterwards. Can you move it just before using xwidget to render the doc?